lib/msal-browser/src/cache/TokenCache.ts

```diff
@@ -109,6 +125,8 @@ export class TokenCache implements ITokenCache {
         } else {
             throw BrowserAuthError.createUnableToLoadTokenError("loadExternalTokens is designed to work in browser environments only.");
         }
+
+        return idTokenHomeAccountId;
```

Comment:
```suggestion
        return idTokenEntity;
```

Reply:
Then do similar with `loadAccessToken`, `loadRefreshToken`, and then build a `CacheRecord` object from those.